src/fluid.cpp

```diff
@@ -74,6 +74,56 @@ bool Fluid::IsGivenKeyMaster(CDynamicAddress inputKey) {
     return false;
 }
 
+/** Checks fluid transactoin operation script amount for invalid values. */
+bool Fluid::CheckFluidOperationScript(const CScript& fluidScriptPubKey, std::string& errorMessage) {
```

Comment:
Similar restrictions must be put onto RPC Interface to prevent reaching this stage in the first place

Reply:
Yes, I agree it is best to stop the transaction before adding it to the wallet and broadcasting/committing it.  

I added CheckFluidOperationScript to CWallet::CreateTransaction to stop the RPC command before committing the transaction.  
![dynamic-fluid-tx-check_20171207](https://user-images.githubusercontent.com/7564876/33751115-529c1692-db9e-11e7-84e1-49e9a230d4da.png)